### PR TITLE
Strict encoding

### PR DESCRIPTION
**Pull request by zkao**

### CODE REVIEW

File: src/consensus/mod.rs [hunk 1]
```diff
@@ -27,6 +27,9 @@ pub mod encode;
 #[doc(hidden)]
 #[macro_use]
 pub mod endian;
+/// implements strict_encode for a few types
+pub mod strict_encode;
+pub use strict_encoding::Error;
```
Comment (h4sh3d) [inline src/consensus/mod.rs:32]: Why do we need `pub use` here?

Comment (zkao) [reply src/consensus/mod.rs]: fixed in commit 1bd883f

Comment (zkao) [reply src/consensus/mod.rs]: however not sure this should be merged to master as it might be too specific to farcaster node

**Review by h4sh3d — APPROVED** [review]
tACK

**Pull request merged**